### ISSUE TITLE
Docker Image build failure

### ISSUE THREAD

**Issue by randomorder** [geosolutions-it/dati-ckan-docker#21]
Docker Image build is currently failing with following error:

```
[91mCannot uninstall 'Markdown'. It is a distutils installed project and thus we cannot accurately determine which files belong to it which would lead to only a partial uninstall.
[0m
Removing intermediate container 1a41763a531f

The command '/bin/sh -c pip install -r "${CKAN_HOME}/src/ckan/requirements.txt"' returned a non-zero code: 1
```

**Comment by tdipisa** [comment]
Also the point below should be updated 

https://github.com/geosolutions-it/dati-ckan-docker/blob/docker-devel/ckan/Dockerfile#L106

since the [#188 Add clean_tags option in harvester](https://github.com/ckan/ckanext-spatial/pull/189) has been merged in ckanext-spatial we can point to the official version now.

**Comment by dydoc** [comment]
the root cause could  be  an interaction between virtaulenv and docker layers corresponding at each RUN command.
The problem seems tricky.
It is advisable to publish an image on Docker Hub when we have a working build.

**Comment by tdipisa** [comment]
fixed via #24
